tests/functional/test_credentials.py

```diff
@@ -1281,6 +1289,34 @@ def test_credential_context_override(self):
             [],
             'e',
         ),
+        # Test case 5: Credentials set with Boto2 config
+        (
+            {},
+            {},
+            [
+                patch(
+                    "botocore.configloader.raw_config_parse",
+                    return_value={
+                        "Credentials": {
+                            "aws_access_key_id": "FAKEACCESSKEY",
+                            "aws_secret_access_key": "FAKESECRETKEY",
+                        }
+                    },
+                ),
+                patch(
+                    "botocore.credentials.ConfigProvider.load",
+                    return_value=None,
+                ),
+                patch(
+                    "botocore.credentials.SharedCredentialProvider.load",
+                    return_value=None,
+                ),
```

Comment:
Is there some environmental factor requiring that these be needed? Removing them passes for me locally?

Reply:
These are more common providers that devs might be using. I'm fine leaving them for now, but not ideal.